mixology/version_solver.py

```diff
@@ -324,10 +324,13 @@ def _next_term_to_try(self):  # type: () -> Optional[Term]
 
         # Prefer packages with as few remaining versions as possible,
         # so that if a conflict is necessary it's forced quickly.
+        # at a tie, the package with least dependencies is chosen
         def _get_min(term):
-            return len(
-                self._source.versions_for(term.package, term.constraint.constraint)
+            versions = self._source.versions_for(
+                term.package, term.constraint.constraint
             )
+            deps = self._source.dependencies_for(term.package, versions[0])
```

Comment:
```suggestion
            deps = self._source.dependencies_for(term.package, versions[0]) if versions else []
```
if no versions available, keep old behaviour